Src/Base/AMReX_DistributionMapping.H

```diff
@@ -159,13 +161,20 @@ class DistributionMapping
 
     static DistributionMapping makeKnapSack   (const MultiFab& weight,
                                                int nmax=std::numeric_limits<int>::max());
+    static DistributionMapping makeKnapSack   (const MultiFab& weight, Real& eff,
+                                               int nmax=std::numeric_limits<int>::max());
     static DistributionMapping makeKnapSack   (const Vector<Real>& rcost,
                                                int nmax=std::numeric_limits<int>::max());
+    static DistributionMapping makeKnapSack   (const Vector<Real>& rcost, Real& eff,
+                                               int nmax=std::numeric_limits<int>::max());
 
     static DistributionMapping makeRoundRobin (const MultiFab& weight);
     static DistributionMapping makeSFC        (const MultiFab& weight, bool sort=true);
+    static DistributionMapping makeSFC        (const MultiFab& weight, Real& eff, bool sort=true);
```

Comment:
It is sort of weird to return the efficiency for `makeSFC` since it doesn't incorporate any communications cost.  Just  a comment.